static/Dockerfile

```diff
@@ -92,14 +93,29 @@ RUN ( \
         && chmod 755 /usr/bin/runc \
     ) || true
 
+# On s390x install docker ce, runc and containerd from docker static s390x build
+RUN ( \
+        [[ "${TARGETPLATFORM}" = 'linux/s390x' ]] \
+        && microdnf install --nodocs tar gzip wget \
+        && curl -L --fail --show-error --silent https://download.docker.com/linux/static/stable/s390x/docker-18.06.3-ce.tgz -o /tmp/docker-18.06.3-ce.tgz \
+        && tar -xzf /tmp/docker-18.06.3-ce.tgz \
```

Comment:
Same as with the dynamic `Dockerfile`. Make sure the tarball is extracted to a specified location.

Reply:
done